puppet/environments/box/modules/profiles/files/box/cli/scripts/Command/Share.php

```diff
@@ -18,14 +17,17 @@ class Share extends Command
 
     protected function configure()
     {
-        $this->setName('share:site')
+        $this->setName('share')
              ->setDescription('Share a local site with a colleague')
```

Comment:
@yiendos The command won't work if you run it through the dashboard's Javascript terminal (or through Cockpit's terminal for example). I don't think we can do anything about this so I suggest to add this to the description. Something like:

```
You need to SSH into the box with `vagrant ssh` first to run this command. This command does not work in the web terminal.
```

I would also explain how it works:

```
This command will start a [ngrok](https://ngrok.com/) session and it will give you a URL to share with your colleagues. Press `ctrl`+`c` to stop ngrok and close the session.
```